packages/web-service/src/pages/common/__tests__/yes-no-conditional.spec.js

```diff
@@ -66,4 +66,31 @@ describe('validator', () => {
       expect(e.details[0].message).toBe("\"yes-conditional-input\" length must be less than or equal to 4000 characters long")
     }
   })
+
+  it('should not throw an error if the user selected yes and the yes-conditional-input validation is skipped', async () => {
+    const payload = { 'yes-no': 'yes', 'yes-conditional-input': '' }
+    const { validator } = await import('../yes-no-conditional.js')
+    const skipValidations = ['yes-conditional-input']
+
+    await expect(validator(payload, skipValidations)).resolves.not.toThrow()
+  })
+
+  it('should not throw an error if the user selected no and the no-conditional-input validation is skipped', async () => {
+    const payload = { 'yes-no': 'no', 'no-conditional-input': '' }
+    const { validator } = await import('../yes-no-conditional.js')
+    const skipValidations = ['no-conditional-input']
+
+    await expect(validator(payload, skipValidations)).resolves.not.toThrow()
+  })
+
+  it('should throw an error if the user selected yes, skipValidations is not defined, and the yes-conditional-input is empty', async () => {
+    try {
+      const payload = { 'yes-no': 'yes', 'yes-conditional-input': '' }
+      const { validator } = await import('../yes-no-conditional.js')
+      await validator(payload)
+    } catch (e) {
+      // eslint-disable-next-line
+      expect(e.details[0].message).toBe("\"yes-conditional-input\" is not allowed to be empty")
```

Comment:
Just a note here -- I realise we're already using `// eslint-disable-next-line` elsewhere to stop it raising linting errors when escaping double quotes in strings so this is just following the existing pattern. But personally this is a convention I'd like us to drop since StandardJS rules are that we should be wrapping strings in single quotes by default, so there's no need to escape double quotes. Happy to keep this here for the time being for consistency though! And at some point in the near future I'll sweep through the repo and remove any such instances.

Reply:
I'll change it. Single quotes are better moving forward.